docs/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ Version 3.42.0-eisop4 (April ?, 2024)
 
 **Implementation details:**
 
+Deprecated the `GenericAnnotatedTypeFactory#addComputedTypeAnnotations(Tree, AnnotatedTypeMirror, boolean)` overload.
+Instead, set `GenericAnnotatedTypeFactory#useFlow` appropriately and
+call `GenericAnnotatedTypeFactory#addComputedTypeAnnotations(Tree, AnnotatedTypeMirror)`.
+Add `GenericAnnotatedTypeFactory#addComputedTypeAnnotationsWithoutFlow(Tree, AnnotatedTypeMirror)` for the old behavior,
```

Comment:
The "old behavior" is not what's important, describe what the method does. I would probably list the new method first and not talk about the field at all.